metomi/rosie/suite_id.py

```diff
@@ -579,7 +579,39 @@ def to_web(self):
 
 def main():
     """Implement the "rose suite-id" command."""
```

Comment:
No command rose suite-id. Does this need to be 
```suggestion
    """Implement the "rosie id" command."""
```

Reply:
Feel free to do this on follow-up if you feel necessary @oliver-sanders. I thought it minor (docstring) so merged anyway.